ldpd/ldpd.h

```diff
@@ -531,13 +532,18 @@ struct kroute {
 };
 
 struct kpw {
+	char                     ifname[IF_NAMESIZE];
 	unsigned short		 ifindex;
 	int			 pw_type;
+	struct in_addr           lsr_id;
 	int			 af;
 	union ldpd_addr		 nexthop;
 	uint32_t		 local_label;
 	uint32_t		 remote_label;
 	uint8_t			 flags;
+	uint32_t                 pwid;
+	char			 vpn_name[L2VPN_NAME_LEN];
+	unsigned short           ac_port_ifindex;
 };
```

Comment:
@rwestphal thanks for your feedback, I will address it soon but a first comment about where to submit the PR to. This is supposed to be related to commit [595b4be](https://github.com/FRRouting/frr/commit/595b4beeb2bb6cbc4f854685cf822becf6158b87) which is in stable/3.0, so I think it should be there. As we talked by e-mail some time ago, with the code in stable/3.0 as it is now,  it will show the PW status as UP by default, even if it's not true.